assets/pages/new-contributions-landing/components/ContributionForm.jsx

```diff
@@ -75,11 +75,9 @@ type PropTypes = {|
   isDirectDebitPopUpOpen: boolean
 |};
 
-type FormValueType = string | null;
-
 // We only want to use the user state value if the form state value has not been changed since it was initialised,
 // i.e it is null.
-const getCheckoutFormValue = (formValue: FormValueType, userValue: FormValueType): FormValueType =>
+const getCheckoutFormValue = <A>(formValue: A | null, userValue: A | null): A | null =>
```

Comment:
why was this change necessary when it looks like we're still only passing in `string` to this function?

Reply:
Good catch, I previously had to check a field of a different type.